src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyLiPClaimantHwFOutcomeHandlerTest.java

```diff
@@ -131,12 +135,76 @@ void shouldNotifyApplicant_HwfOutcome_NoRemission_Hearing() {
             // Then
             verify(notificationService, times(1)).sendMail(
                 EMAIL,
-                EMAIL_TEMPLATE_HWF,
+                EMAIL_TEMPLATE_NO_REMISSION,
                 getNotificationDataMapNoRemissionHearing(),
                 REFERENCE_NUMBER
             );
         }
 
+        @Test
+        void shouldNotifyApplicant_HwfOutcome_RefNumberUpdated_ClaimIssued() {
+            // Given
+            HelpWithFeesDetails hwfeeDetails = HelpWithFeesDetails.builder()
+                .hwfCaseEvent(UPDATE_HELP_WITH_FEE_NUMBER)
+                .build();
+            CaseData caseData = CaseDataBuilder.builder().atStateClaimSubmitted().build().toBuilder()
```

Comment:
Reuse Case Data from as created in recent changes. Please check Pablo's test cases.